arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -2700,16 +2589,144 @@ ReplicationApplier* RestReplicationHandler::getApplier(bool& global) {
   }
 }
 
-//////////////////////////////////////////////////////////////////////////////
-/// @brief condition locker to wake up holdReadLockCollection jobs
-//////////////////////////////////////////////////////////////////////////////
+Result RestReplicationHandler::createBlockingTransaction(aql::QueryId id,
+                                                         LogicalCollection& col,
+                                                         double ttl) const {
+  // This is a constant JSON structure for Queries.
+  // We do not need nodes or variables.
+  // We only need collections, with corresponding lock type
+  auto planBuilder = std::make_shared<VPackBuilder>();
+  planBuilder->openObject();
```

Comment:
The comment above says "we only need collections, with corresponding lock type", but no collection is ever added to the planBuilder. So the comments are at least misleading.